gdUnit.bat

```diff
@@ -0,0 +1,17 @@
+@echo off
+cls
+SET GODOT_BIN=D:\develop\Godot.exe
+
+ECHO %GODOT_BIN%
+
+if not defined GODOT_BIN (
+	echo "GODOT_BIN is not set."
+	echo "Please set the environment variable 'SET GODOT_BIN=<path to godot.exe>'"
+	exit /b -1
+)
+
+echo "test"
+
+%GODOT_BIN% -s -d .\addons\gdUnit3\GdUnitCmdTool.gd %*
+
+exit /b
```

Comment:
My .bat is a little rusty but

- I prefer all uppercase for for DOS commands
- Missing new line at end
- Can't we use `.env` file to find Godot (long shot :-/)

Reply:
here is the first draft of tool documentation ;) is not finish and public yet
https://github.com/MikeSchulze/gdUnit3/wiki/Command-Line-Tool

Thanks for review my change set ;) 
I will check your comments and apply 

`My .bat is a little rusty but` hehe yes i had this in my mind since i started on windows. long time ago i wrote batch files
After short read the recomended extention is 'cmd'